Sources/Build/BuildDescription/SwiftModuleBuildDescription.swift

```diff
@@ -521,7 +521,10 @@ public final class SwiftModuleBuildDescription {
 
         // Only add the build path to the framework search path if there are binary frameworks to link against.
         if !self.libraryBinaryPaths.isEmpty {
-            args += ["-F", self.buildParameters.buildPath.pathString]
+            args += [
```

Comment:
question: Does this `if` block and the one [here](https://github.com/swiftlang/swift-package-manager/pull/7998/files?w=1#diff-2affeac76f1ef6916e6d06095d4e7f679e592c07c2a7556ad4eaddddd441ed91R664) always need to match? If so, can we refactor by moving the common code into its own function so that updating in one place will automatically update all usage?

Reply:
I think we should. I'll refactor it to a small helper function here.